src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -43,12 +45,43 @@
      */
     private static final String MAX = "max";
 
+    /**
+     * Map that mapping between a check and its ancestor token types.
```

Comment:
`Map that mapping between a check and its ancestor token types.`
-->
`Mapping between a check and its ancestor token types.`

Reply:
done